.github/workflows/actions/manage-doppler-secret/manage.sh

```diff
@@ -4,17 +4,18 @@
 DOPPLER_PROJECT=$1
 NAMESPACE=$2
 ENVIRONMENT=$3
+TENANT_NAME=$4
 DOPPLER_OPERATOR_SECRET_NAME="${NAMESPACE}-${DOPPLER_PROJECT//_/-}"
```

Comment:
Is this also DOPPLER_OPERATOR_SECRET_NAME  going to be a problem?

Reply:
Technically it shouldn't since this is the kubernetes secret name. We only have one `shared` namespace and one independent doppler-operator-system per cluster. They would all have the same name across clusters but the doppler-operator-systems' don't communicate with each other across clusters. It simply uses the `serviceToken` value to determine which branch config to reference, which is what we did change. If you look at any of the tokens, you can see that the prefix is what determines which branch config to reference (e.g. `dp.st.uat_faut-01.EU2exxxxxxx`). The kubernetes secret name is just that, a unique secret name in the doppler-operator-system namespace. However, the more I think about it, it's probably best to be more descriptive instead of less. Changed here: https://github.com/onboardiq/gh-common-workflows/pull/16/commits/7a37f0708cf6d9faca9c4799552774373f8c5ea2